(DOCS-2790) incident mobile widget (#12432)
DataDog/documentation: content/en/mobile/_index.md

## Widgets

### Open Incidents widget

> {{< img src="mobile/incident_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}

Reviewer: ```suggestion
{{< img src="mobile/incident_widget.png" alt="Datadog incident mobile widget displayed on Android and iOS devices" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
```

View your [open incidents][13] from your mobile home screen with Datadog widgets.

> You can tap any open incident displayed in the widget to have it open with more details in the Datadog mobile app - perfect for diving deeper into issues. 

Reviewer: ```suggestion
To dive deeper into issues, tap any open incident displayed in the widget to have it open with more details in the Datadog mobile app. 
```

> Also, you can customize your Open Incidents widgets by filtering on:

Reviewer: (Just adding a NL here to make sure markdown plays nice in the future)
```suggestion
Also, you can customize your Open Incidents widgets by filtering on:

```

- Organization
- Serverity levels
- Customers impacted
- Ordering

#### Create an Open Incidents widget

{{< tabs >}}
{{% tab "iOS" %}}

> - Long press on your home screen.
> - Tap the “+” button on the top left corner of the screen.
> - Search for "Datadog" widgets.
> - Select your prefered size (small, medium, or large).
> - Drag the widget to your desired, on-screen location.

Reviewer: ```suggestion
1. Tap the **+** button on the top left corner of the screen.
2. Search for "Datadog" widgets.
3. Select your preferred size (small, medium, or large).
4. Drag the widget to your desired, on-screen location.
```

{{% /tab %}}
{{% tab "Android" %}}

> - Long press on your home screen.
> - Tap the "Widgets" button on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
> - Drag the widget to your desired, on-screen home screen.
> - Resize the widget to fit your preference.

Reviewer: ```suggestion
1. Long press on your home screen.
2. Tap the **Widgets** button on your home screen editor. If you have app shortcuts, it might appear as only an icon on the top right corner of the bubble.
3. Drag the widget to your desired, on-screen home screen.
4. Resize the widget to fit your preference.
```

{{% /tab %}}
{{< /tabs >}}

#### Edit an Open Incidents widget

{{< tabs >}}
{{% tab "iOS" %}}

- Long press on the widget to configure.
- Tap "Edit Widget."
- Tap "Choose" next to the Organzation label to fetch open incidents from the selected organization.
- Tap "SEV-1 and SEV-2" next to the Severities label to specify severity filters.
- Tap "Both" next to the Customer Impacted label to filter on open incidents that have impacted customers.
- Type in the "Type additional filters" text box to specify any further filtering.

> - Tap "Ordering" to specify the order of how incidents will be listed.

Reviewer: ```suggestion
- Tap "Ordering" to specify the order of how incidents are listed.
```

> - Long press on the widget to configure.
> - Tap "Edit Widget."
> - Tap "Choose" next to the Organzation label to fetch open incidents from the selected organization.
> - Tap "SEV-1 and SEV-2" next to the Severities label to specify severity filters.
> - Tap "Both" next to the Customer Impacted label to filter on open incidents that have impacted customers.
> - Type in the "Type additional filters" text box to specify any further filtering.
> - Tap "Ordering" to specify the order of how incidents will be listed.
> - Tap out of the widget to validate your selection and exit the configuration screen.

Reviewer: ```suggestion
1. Long press the widget to configure.
2. Tap **Edit Widget**.
2. Tap **Choose** next to the **Organzation** label to fetch open incidents from the selected organization.
3. Tap **SEV-1 and SEV-2** next to the Severities label to specify severity filters.
4. Tap **Both** next to the **Customer Impacted** label to filter on open incidents that have impacted customers.
5. Type in the **Type additional filters** text box to specify any further filtering.
6. Tap **Ordering** to specify the order of how incidents are listed.
7. Tap outside of the widget to save your selection and exit the configuration screen.
```

{{% /tab %}}
{{% tab "Android" %}}

> - Tap the widget title to configure.
> - Tap "Organization" to fetch open incidents from the selected organization.
> - Tap "Severities" to specify severity filters.
> - Tap "Customer impacted" to filter on open incidents that have impacted customers.
> - Tap "Query" to specify any further filtering.
> - Tap "Sorted by" to specify the order of how incidents will be listed.
> - Tap "Save" or "Apply" to validate your selection and exit the configuration screen.
> - Long press and resize the widget to fit your preference.

Reviewer: ```suggestion
1. Tap the widget title to configure.
2. Tap **Organization** to fetch open incidents from the selected organization.
3. Tap **Severities** to specify severity filters.
4. Tap **Customer impacted** to filter on open incidents that have impacted customers.
5. Tap **Query** to specify any further filtering.
6. Tap **Sorted by** to specify the order of how incidents are listed.
7. Tap **Save** or **Apply** to save your selection and exit the configuration screen.
8. Long press and resize the widget to fit your preference.
```

{{% /tab %}}
{{< /tabs >}}

#### Display Open Incidents from multiple organizations

You can display open incidents from multiple organizations on your mobile home screen. 

{{< tabs >}}
{{% tab "iOS" %}}

> - Tap "Choose" next to the Organzation label to fetch open incidents from the selected organization.

Reviewer: ```suggestion
- Tap **Choose** next to the Organization label to fetch open incidents from the selected organization.
```

{{% /tab %}}
{{% tab "Android" %}}

> - Tap the widget title to configure.
> - From the configuration screen, tap "Organization".
> - Select a new organization (You might need to sign in).
> - Size the widget to fit your preference.
> - Tap "Save" or "Apply". 

Reviewer: ```suggestion
1. Tap the widget title to configure.
2. From the configuration screen, tap **Organization**.
3. Select a new organization (you may need to sign in).
4. Size the widget to fit your preference.
5. Tap **Save** or **Apply**. 
```

{{% /tab %}}
{{< /tabs >}}

#### Delete an Open Incidents widget

{{< tabs >}}
{{% tab "iOS" %}}

> Delete a widget by tapping the "-" button at the top left of the widget when editing your home screen, or by long pressing on the widget and selecting "Remove Widget".

Reviewer: ```suggestion
Delete a widget by tapping the **-** button at the top left of the widget when editing your home screen, or by long pressing the widget and selecting **Remove Widget**.
```

{{% /tab %}}
{{% tab "Android" %}}

> Delete a widget by long pressing, dragging, and dropping the widget to the "Remove" button.

Reviewer: ```suggestion
Delete a widget by long pressing, dragging, and dropping the widget to the **Remove** button.
```

{{% /tab %}}
{{< /tabs >}}

### SLOs widget

> {{< img src="mobile/slo_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}

Reviewer: Can we please update the alt text on this to be more accessible and descriptive while we're updating this page?
```suggestion
{{< img src="mobile/slo_widget.png" alt="Application Uptime SLO widgets displayed on Android and iOS devices" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
```

### Monitors widget

> {{< img src="mobile/monitor_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}

Reviewer: ```suggestion
{{< img src="mobile/monitor_widget.png" alt="Configured monitor widgets displayed on Android and iOS screens" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
```

> View your [monitors][15] from your home screen with Datadog widgets. Tap on any cell to open the "Monitor Search" screen in the app, with your monitors already filled in.

Reviewer: ```suggestion
View your [monitors][15] from your home screen with Datadog widgets. Tap any cell to open the **Monitor Search** screen in the app, with your monitors already filled in.
```

{{< img src="mobile/shortcut_shadow.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Quick Actions">}}

> Long-press on the app icon to surface a quick-action sheet of your top five [Frequently Viewed By Me][16] dashboards for iOS (measured by view count and recency) and your five most opened dashboards on mobile for Android. Press on a result to open the dashboard in-app.

Reviewer: ```suggestion
Long-press the app icon to sdisplay a quick-action sheet of your top five [Frequently Viewed By Me][16] dashboards for iOS (measured by view count and recency), or your five most opened dashboards on mobile for Android. Tap a result to open the dashboard in-app.
```

{{< img src="mobile/siri_shadow.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Shortcuts">}}

> For more information on Siri Shortcuts and Suggestions, refer to the [Apple Siri Documentation][17].

Reviewer: ```suggestion
For more information about Siri shortcuts and suggestions, read the [Apple Siri documentation][17].
```

- Have Wi-Fi enabled
- Have Handoff enabled

> For more information on Handoff, refer to the [Apple Handoff Documentation][18].

Reviewer: ```suggestion
For more information about Handoff, read the [Apple Handoff documentation][18].
```
